src/components/Lookup/styled/chip.js

```diff
@@ -6,6 +6,12 @@ const StyledChip = attachThemeAttrs(styled(Chip))`
     flex: 1 1 auto;
     margin: 2px;
     background-color: ${props => props.palette.background.main};
+    width: 0px;
+
+    & > span {
+        min-width: 0px;
+        flex-basis: 99%;    
+    }
```

Comment:
this is because changing the value directly in the Structured component... a lot of components are affected .. so better to change it here to only affect the Chip associated with the lookup value